index.html

```diff
@@ -151,19 +157,17 @@ <h3 id="noharm" data-export="" data-dfn-type="dfn">
   </h3>
   <p>
 When we are adding a feature or technology to the web,
-we will consider what harm it could do to society or groups,
+we will work to prevent or mitigate any harm it might cause society or groups,
 especially to vulnerable people.
+We consider a range of threat models that account for abuse scenarios
+at different scales, from societal to interpersonal.
 We will prioritize potential benefits for web users over potential benefits to web developers,
 content providers, user agents, advertisers or others in the ecosystem,
 in line with the
 <a data-cite="design-principles#priority-of-constituencies">priority of constituencies</a>.
-We will ensure the requirements and views of marginalized communities
-and underrepresented groups are heard and respected.
-We will build new web technologies in a collaborative manner
-according to open processes
-(for example, the <a href="https://www.w3.org/Consortium/Process/">W3C process</a>),
-and adhering to codes of conduct
-(such as the W3C <a href="https://www.w3.org/Consortium/cepc/">Code of Ethics and Professional Conduct</a>).
+We will actively seek to involve marginalized communities
+in the design of features and technologies,
+and ensure the views of underrepresented groups are heard and respected.
```

Comment:
```suggestion
We commit to making the effort to learn and understand diverse perspectives,
and will strive to reflect a respect for that diversity in the designs we produce,
so that our designs properly respect the interests and views of
all of the people who might be affected by them.
```

Reply:
This is more concise, but hopefully still captures the ideas in Martin's comments.